launch/launch/actions/group_action.py

```diff
@@ -34,39 +35,72 @@
 @expose_action('group')
 class GroupAction(Action):
     """
-    Action that yields other actions, optionally scoping launch configurations.
+    Action that yields other actions, optionally scoping and forwarding launch configurations.
 
     This action is used to nest other actions without including a separate
     launch description, while also optionally having a condition (like all
-    other actions), scoping launch configurations, and/or declaring launch
-    configurations for just the group and its yielded actions.
+    other actions), scoping launch configurations, forwarding launch
+    configurations, and/or declaring launch configurations for just the
+    group and its yielded actions.
+
+    When scoped is set to True, changes to launch configurations are
+    limited to the scope of actions in the group action.
+
+    When scoped is set to True and forwarded is set to True, all existing
+    launch configurations are available in the scoped context.
+
+    When scoped is set to True and forwarded is set to False, all existing
+    launch configurations are removed from the scoped context unless it is
+    explicitly forwarded by adding its key to the launch_configurations
+    dictionary with a None value.
```

Comment:
This feels weird to me, maybe this is what you were talking about when @hidmic suggested deduplicating the two lists. I'll wait for feedback from others before saying what I think we should do (not sure yet).

Reply:
Weirdness hopefully resolved with the new `ResetLaunchConfigurations` action which does not use special flags (ie None) to indicate a pass through.